Makefile

```diff
@@ -337,11 +371,15 @@ standalone: node_modules \
             $(LODASH_OBJS) \
             $(built_standalone_js_files) \
             $(built_standalone_shared_js_files) \
-            $(built_standalone_l10n_files)
+            $(built_standalone_l10n_files) \
+						$(BUILT)/standalone/webpack.config.js \
+						$(BUILT)/standalone/content/webappEntryPoint.js
```

Comment:
These 2 lines are redundant as `standalone_misc_files` includes them for `built_standalone_js_files`